test/CSharp/CodeCracker.Test/Design/CatchEmptyTests.cs

```diff
@@ -106,13 +107,13 @@ public void Foo()
                 {
                     // do something
                 }
-                catch
+                [|catch
                 {
                     if (x == 1)
                         throw;
                     else
                         return;
-                }
+                }|]
```

Comment:
💡 The new test library revealed that the span of this diagnostic covers the entire catch clause. If this wasn't specifically intended, consider filing a bug to only report the warning on the `catch` token.